library/kani/src/lib.rs

```diff
@@ -7,11 +7,10 @@
 // Used for rustc_diagnostic_item.
 // Note: We could use a kanitool attribute instead.
 #![feature(rustc_attrs)]
-// This is required for the optimized version of `any_array()`
-#![feature(generic_const_exprs)]
-#![allow(incomplete_features)]
 // Used to model simd.
 #![feature(repr_simd)]
+#![feature(generic_const_exprs)]
```

Comment:
Is this a clippy cleanup?

Reply:
No. I updated it because before it was needed for `simd` and any_array. However, now it is only needed for `simd`.